ocs_ci/utility/utils.py

```diff
@@ -922,3 +922,11 @@ def get_csi_versions():
             version = container['image'].split("/")[-1].split(":")[1]
             csi_versions[name] = version
     return csi_versions
+
+
+def create_directory_path(path):
+    """
+    Creates directory if path doesn't exists
+    """
+    if not os.path.exists(path):
```

Comment:
Should we log something in the event the directory already exists? Right now it will just move on silently.

Reply:
Done